web/templates/invite_user_modal.hbs

```diff
@@ -40,16 +40,16 @@
                 <option {{#if this.default }}selected{{/if}} name="expires_in" value="{{this.value}}">{{this.description}}</option>
             {{/each}}
         </select>
-        <p id="expires_on"></p>
-        <div id="custom-invite-expiration-time" class="dependent-settings-block">
+        <p class="input-time-valid-till"></p>
+        <div id="custom-invite-expiration-time" class="dependent-settings-block custom-time-input-container">
             <label for="expires_in" class="modal-field-label">{{t "Custom time" }}</label>
-            <input type="text" autocomplete="off" name="custom-expiration-time" id="custom-expiration-time-input" class="custom-expiration-time inline-block" value="" maxlength="3"/>
-            <select class="custom-expiration-time invite-user-select modal_select bootstrap-focus-style" id="custom-expiration-time-unit">
+            <input type="text" autocomplete="off" name="custom-expiration-time" id="custom-expiration-time-input" class="custom-input-time-value inline-block" value="" maxlength="3"/>
+            <select class="custom-input-time-unit invite-user-select modal_select bootstrap-focus-style" id="custom-expiration-time-unit">
```

Comment:
I think it would be good to be consistent with either "custom-time-input" or "custom-input-time" ... I lean towards the first option. That will make a git-grep for these class names be more useful.

"'custom-time-input-container", "custom-time-input-unit", "custom-time-input-value", etc.